osx/make_app.sh

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env bash
+
+# Usage: ./setup.py bdist_app
+# $1: wheel name
+# $2: package name
+set -e
+
+. ./utils/functions.sh
+
+python='python3'
+osx_dir=$(dirname "$0")
+plover_dir=$(dirname $osx_dir)
+plover_wheel=$plover_dir/dist/$1.whl
```

Comment:
It's better to quote variables to avoid unwanted expansion by the shell:
```bash
> mkdir 'unsafe name unsafe*'
> echo 'osx_dir=$(dirname "$0"); ls $osx_dir' > unsafe\ name\ unsafe\*/test.sh
> bash -x ./unsafe\ name\ unsafe\*/test.sh
++ dirname './unsafe name unsafe*/test.sh'
+ osx_dir='./unsafe name unsafe*'
+ ls ./unsafe name 'unsafe name unsafe*'
ls: cannot access './unsafe': No such file or directory
ls: cannot access 'name': No such file or directory
'unsafe name unsafe*':
test.sh
```